### PR TITLE
added How to create Excel file from SharePoint list

### PR DESCRIPTION
**Pull request by PetrBushuev**

### CODE REVIEW

File: source/flow/how-tos/documents/create-excel-file-from-sharepoint-list.rst [hunk 1]
```diff
@@ -0,0 +1,184 @@
+.. title:: How to create excel file from sharepoint list in Power Automate (Microsoft Flow)
+
+.. meta::
+   :description: create excel file from sharepoint list in Power Automate (Microsoft Flow)
```
Comment (MaxAnshin) [inline source/flow/how-tos/documents/create-excel-file-from-sharepoint-list.rst:4]: Too short a description. Two reasons to rewrite it.
1. We usually write in the description something important that describes to users what about the article. 
2. This description shows in the google search. With that short description, we will take a small place in the google search rank. The bigger place we take in google search, the more chances users will go to our website. 

Please, write here sentence with between 200 - 300 symbols. You can use the intro from the article.

File: source/flow/how-tos/documents/create-excel-file-from-sharepoint-list.rst [hunk 2]
```diff
@@ -0,0 +1,184 @@
+.. title:: How to create excel file from sharepoint list in Power Automate (Microsoft Flow)
+
```
Comment (MaxAnshin) [inline source/flow/how-tos/documents/create-excel-file-from-sharepoint-list.rst:2]: @PetrBushuev I've reviewed the article. Good work! Please take a look at comments i left. 
One more general comment: Please replace pictures with shadows on borders to pictures without them. I prepared [the folder](https://plumsailteam.sharepoint.com/:f:/s/actions/EmoU89TwH6tJkRlqjf8WR1MBO4sg_pqMABDoxcjgTKTULw?e=BFSmnm) where you can find pictures without shadows on borders.

File: source/flow/how-tos/documents/create-excel-file-from-sharepoint-list.rst (hunk 1 above)
Comment (cherinaa) [reply source/flow/how-tos/documents/create-excel-file-from-sharepoint-list.rst:4]: Ok. @PetrBushuev Let's rewrite the meta description. Please, take this part of the article for it:

> We will populate an Excel template with data from the SharePoint list. The case is the following: we add a new order to the SharePoint list, this data fills an Excel template, then we send the resulting document by email. First, we will prepare an XLSX template in Plumsail Documents.

File: source/flow/how-tos/documents/create-excel-file-from-sharepoint-list.rst [hunk 3]
```diff
@@ -0,0 +1,184 @@
+.. title:: How to create excel file from sharepoint list in Power Automate (Microsoft Flow)
```
[the remark this answers is not in the document]
Comment (cherinaa) [reply source/flow/how-tos/documents/create-excel-file-from-sharepoint-list.rst:1]: @PetrBushuev please, correct.

File: source/flow/how-tos/documents/create-excel-file-from-sharepoint-list.rst [hunk 4]
```diff
@@ -0,0 +1,184 @@
...
+Upload and prepare Excel document template
+-------------------------------------------
+
+First, we will prepare a document template in Plumsail Documents.
+
+`Create a free account <https://auth.plumsail.com/account/register?returnurl=https://account.plumsail.com/documents/processes/reg>`_ for Plumsail Documents. 
+In the account, add a new document generation process. 
+There are two options: create it from a template or a blank.
+
+.. image:: ../../../_static/img/flow/how-tos/8485-create-process.png
```
[the remark this answers is not in the document]
Comment (cherinaa) [reply source/flow/how-tos/documents/create-excel-file-from-sharepoint-list.rst:28]: Sure, there is no need to show this picture without the button "Use this template". @PetrBushuev Let's remove it and reorganize the text a bit. Move this sentence up. 

image

Replace the picture of adding a new process with a picture showing the template like this: 
image

File: source/flow/how-tos/documents/create-excel-file-from-sharepoint-list.rst [hunk 5]
```diff
@@ -0,0 +1,184 @@
...
+
+
+Create Excel file from SharePoint list in Power Automate flow
+=============================================================
+
+This article demonstrates how to populate an Excel template with data from a SharePoint list in `Power Automate (Microsoft Flow) <https://flow.microsoft.com>`_. 
+
+The case is the following: we add a new order to the list, this data fills a document template, then we send the resulting document by email.
+
+.. contents::
```
Comment (cherinaa) [inline source/flow/how-tos/documents/create-excel-file-from-sharepoint-list.rst:14]: @PetrBushuev Please, add a name to the Table of contents

File: source/flow/how-tos/documents/create-excel-file-from-sharepoint-list.rst [hunk 6]
```diff
@@ -0,0 +1,184 @@
...
+After configuring deliveries, go to the Power Automate account.
+
+
+Prepare SharePoint list
+-------------------------
+
+In our example, one purchase order is one item in the SharePoint list. 
+Since the order may include several products or services, one column in the row contains a table. 
+We used `Plumsail Forms for SharePoint <https://plumsail.com/forms/sharepoint-forms/>`_ to adjust the list to our needs. 
+Your case might be different without data tables, so you can skip this chapter and go to step №4.
```
Comment (cherinaa) [inline source/flow/how-tos/documents/create-excel-file-from-sharepoint-list.rst:94]: We didn't enumerate steps here in the article, that's why 'go to step №4' looks weird. You can just say something like this: .. you can skip this and use your SharePoint list as is.

**Review by cherinaa — CHANGES_REQUESTED** [review]
@PetrBushuev Please, review the comments and apply corrections.

**Pull request merged**